components/brave_sync/brave_sync_prefs.cc

```diff
@@ -10,7 +10,7 @@
 #include "base/base64.h"
 #include "base/logging.h"
 #include "components/os_crypt/os_crypt.h"
-#include "components/pref_registry/pref_registry_syncable.h"
+#include "components/prefs/pref_registry_simple.h"
```

Comment:
can we remove the dep for `//components/pref_registry`?

Reply:
Yes, indeed. Removed in 5a5c2bc6f4df05ec0ac8e4f6f5227c8f0247bf6a